codecarbon/emissions_tracker.py

```diff
@@ -23,8 +23,8 @@
 from codecarbon.input import DataSource
 from codecarbon.output import BaseOutput, EmissionsData, FileOutput
 
-logging.basicConfig(level=os.environ.get("CODECARBON_LOGLEVEL", "WARN"))
 logger = logging.getLogger(__name__)
+logger.setLevel(level=os.environ.get("CODECARBON_LOGLEVEL", "WARN"))
```

Comment:
can we update this in the documentation for a user to set Logging Level by setting `CODECARBON_LOGLEVEL` environment variable